test/browser.rollbar.test.js

```diff
@@ -477,6 +477,39 @@ describe('options.captureUncaught', function() {
     done();
   });
 
+    it('should capture exta frames when stackTraceLimit is set', function(done) {
```

Comment:
indentation is off a little bit here, but small potatoes ;)

Reply:
Oops. I'll fix it.